src/com/sap/piper/DefaultValueCache.groovy

```diff
@@ -0,0 +1,27 @@
+package com.sap.piper
+
+class DefaultValueCache implements Serializable {
+    private static DefaultValueCache instance
+
+    private Map defaultValues
+
+    private DefaultValueCache(Map defaultValues){
+        this.defaultValues = defaultValues
+    }
+
+    static getInstance(){
+        return instance
+    }
+
+    static createInstance(Map defaultValues){
```

Comment:
We are in a strong singe threaded world, I suppose? Otherwise we have to synchronize the method.

Reply:
I guess this is OK. We should be in a single thread per job execution ...